sizereport.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+	repo: 'developit/preact',
+	path: ['dist/*', 'compat/dist/*', 'debug/dist/*', 'hooks/dist/*'],
```

Comment:
Maybe we should exclude `.map` files? Or be specific by adding `.js` as extension?
Looking into travis at what it generates, seems to report the diff for the `.map` as well. Not sure if we should _track_ those as well.

Reply:
fixed in https://github.com/developit/preact/pull/1633/commits/51dbc5fcdf04f3943fae59bac28ba8f54af5544b